examples/tutorials/text.py

```diff
@@ -0,0 +1,139 @@
+"""
+Plotting text
+=============
+
+It is often useful to add annotations to a map plot. This is handled by
+:meth:`pygmt.Figure.text`.
+"""
+
+import os
+import pygmt
+
+###############################################################################
+# Basic map annotation
+# --------------------
+#
+# Text annotations can be added to a map using the :meth:`text` method of the
+# :class:`pygmt.Figure`.
+#
+# Here we create a simple map and add an annotation using the ``text``, ``x``,
+# and ``y`` arguments to specify the annotation text and position in the
+# projection frame. ``text`` accepts 'str' types, while ``x``, and ``y``
+# accepts either 'int' or 'float' types.
+
+fig = pygmt.Figure()
+with pygmt.config(MAP_FRAME_TYPE="plain"):
+    fig.basemap(region=[108, 120, -5, 8], projection="M20c", frame="a")
```

Comment:
As an introductory tutorial, we probably should avoid using unrelated methods `pygmt.config()`, although plain frames sometimes look better than "fancy" frames. But I'm also OK to keep `pygmt.config()` here.

Reply:
I think it's ok to keep it here for now, this might reduce the number of 'fancy' frames out there in the world :laughing: